### ISSUE TITLE
Electron remote DB sync failure, error status "aborting" with no errors listed

### ISSUE THREAD

**Issue by brettins** [pouchdb/pouchdb#8028]
### Issue
PouchDB fails to sync to the CouchDB on Electron distributed package, Windows 10. 

It works fine on a development/debug build. However, when syncing the databases on a packaged Windows.exe it gets the following error:

```
[tt [Error]] {
  result: {
    ok: false,
    start_time: '2020-03-11T20:01:04.542Z',
    docs_read: 0,
    docs_written: 0,
    doc_write_failures: 0,
    errors: [],
    status: 'aborting',
    end_time: '2020-03-11T20:01:06.053Z',
    last_seq: 0
  }

```

CouchDB logs don't seem to indicate any fatal errors:

```
[notice] 2020-03-11T17:10:53.170083Z couchdb@127.0.0.1 <0.8694.35> 0165a0f16d [couchdb_url]:5984 40.127.0.230 admin PUT /_users/org.couchdb.user:f751306f-cb2b-4e32-b3e4-9169fd9a034f 201 ok 50
[error] 2020-03-11T17:10:53.191724Z couchdb@127.0.0.1 <0.8764.35> b4f25e8b76 Request to create N=3 DB but only 1 node(s)
[notice] 2020-03-11T17:10:53.320152Z couchdb@127.0.0.1 <0.8764.35> b4f25e8b76 [couchdb_url]:5984 40.127.0.230 admin PUT /analyticsdb_f751306f-cb2b-4e32-b3e4-9169fd9a034f 201 ok 141
[notice] 2020-03-11T17:10:53.343778Z couchdb@127.0.0.1 <0.8771.35> 074bea1045 [couchdb_url]:5984 40.127.0.230 admin PUT /analyticsdb_f751306f-cb2b-4e32-b3e4-9169fd9a034f/_security 200 ok 20
[notice] 2020-03-11T17:10:54.072009Z couchdb@127.0.0.1 <0.8645.35> 3ed7f08467 [couchdb_url]:5984 209.161.245.101 f751306f-cb2b-4e32-b3e4-9169fd9a034f GET /analyticsdb_f751306f-cb2b-4e32-b3e4-9169fd9a034f/ 200 ok 3
[notice] 2020-03-11T17:10:54.731104Z couchdb@127.0.0.1 <0.8842.35> e0bbae4512 [couchdb_url]:5984 209.161.245.101 f751306f-cb2b-4e32-b3e4-9169fd9a034f GET / 200 ok 1
[notice] 2020-03-11T17:10:54.744402Z couchdb@127.0.0.1 <0.8844.35> 7fc5d34084 [couchdb_url]:5984 209.161.245.101 f751306f-cb2b-4e32-b3e4-9169fd9a034f GET / 200 ok 0
[notice] 2020-03-11T17:10:55.371486Z couchdb@127.0.0.1 <0.9016.35> 9995ef7731 [couchdb_url]:5984 209.161.245.101 f751306f-cb2b-4e32-b3e4-9169fd9a034f GET /analyticsdb_f751306f-cb2b-4e32-b3e4-9169fd9a034f/_local/k3JedoHg564SqtiPpdsIVw%3D%3D? 404 ok 1
```

### Info
- Environment: Node.js
- Platform: Electron, Windows 10
- Adapter: None specified, pouch-db node used
- Server: CouchDB

--------
I've looked at the following related issues:
https://github.com/pouchdb/pouchdb/issues/7836
https://github.com/pouchdb/pouchdb/issues/2377
https://github.com/pouchdb/pouchdb/issues/7588

The latter of which seems to have an intriguing note about btoa or atob error they managed to get, though they don't specify how they got the error to appear.  The other two I wasn't able to use or figure out how it would be relevant to my case.

**Comment by knubie** [comment]
@brettins Any information in the Network tab of developer tools? E.g. response data, etc.

**Comment by brettins** [comment]
Is there a way to pull that up on a release version of Electron?

**Comment by knubie** [comment]
@brettins Yeah, see `toggleDevTools` here: https://www.electronjs.org/docs/api/menu-item

**Comment by brettins** [comment]
Hrm, my calls are in the main class, not the renderer, and I think happen before the renderer actually loads. These things reference menus / Windows to show dev tools.

**Comment by fractalfederation** [comment]
im having same issue on quasar electron with pouchdb. all DBs sync except the one with binary blobs as attachments. im using pouchdb 7.2.1. using electron 8.2.2. im running quasar build -m electron and using the linux executable on ubuntu. the btoa atob issue mentioned by OP was supposedly fixed and committed and is included in 7.2.1?
